internal/kv/csv_file.go

```diff
@@ -56,10 +57,26 @@ type kvCSVFile struct {
 	// This is optional and defaults to using the first line of the CSV file as the
 	// header.
 	Header string `json:"header"`
-	mu     sync.Mutex
+	mu     *sync.Mutex
```

Comment:
style: why the switch to a pointer? the style guide recommends not using pointers and this introduces some risk if the struct is ever copied (copies share mutex if it's a pointer).

Reply:
I did this to prevent the copy of a lock as it was returned from the constructor, a better fix is returning a pointer type from that constructor so I shifted to that instead. 👍